docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -2911,6 +2911,124 @@ Configuration of a new violation can use the following trigger types:
+
+One way of accessing encapsulated traffic at the destination host is through a software called RCDCAP, which is a daemon that creates a virtual interface on which both GRE and ERSPAN headers are decapsultated prior to the traffic being injected on the configured interface. Security Onion can then feed on that interface like it would on any other.
+
+
+Assomptions for the example:
+The switch is at IP 10.10.10.1, the monitored switch port is GigabitEthernet10 and the Security Onion monitoring destination IP is 10.10.10.10 on eth2, ideally on a seperate interface optimized for monitoring.
+
+On Security Onion:
+
+ * Enable Inverse repository for Security Onion:
+
+  sudo bash -c 'cat << EOL >/etc/apt/sources.list.d/securityonion-inverse.list
+  deb http://inverse.ca/downloads/PacketFence/securityonion trusty trusty
+  EOL'
+
+  gpg --keyserver keyserver.ubuntu.com --recv 19CDA6A9810273C4
+  gpg --export --armor 19CDA6A9810273C4 | sudo apt-key add -
+
+ * Install rcdcap
+
+  sudo apt-get update
+  sudo apt-get install rcdcap  
+
+ * Create the file /etc/init.d/rcdcap with a similar content:  
+----
+#!/bin/sh
+### BEGIN INIT INFO
+# Provides:          rcdcap
+# Required-Start:    $local_fs $network $named $time $syslog
+# Required-Stop:     $local_fs $network $named $time $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Description:       rcdcap - remote capture preprocessor
+### END INIT INFO
+
+PIDFILE=/var/run/rcdcap.pid
+
+start() {
+  if [ -f /var/run/$PIDNAME ] && kill -0 $(cat /var/run/$PIDNAME); then
+    echo 'Service already running' >&2
+    return 1
+  fi
+  echo 'Starting service…' >&2
+  local CMD="$SCRIPT &> \"$LOGFILE\" & echo \$!"
+  rcdcap -i eth2 --erspan --tap-persist --tap-device mon1 -d
+  pidof rcdcap > "$PIDFILE"
+  return 0
+  echo 'Service started' >&2
+}
+
+stop() {
+  if [ ! -f "$PIDFILE" ]; then
+    echo 'Service not running' >&2
+    return 1
+  fi
+  echo 'Stopping service…' >&2
+  kill -9 $(cat "$PIDFILE") && rm -f "$PIDFILE"
+  echo 'Service stopped' >&2
+}
+
+
+case "$1" in
+  start)
+    start
+    ;;
+  stop)
+    stop
+    ;;
+  restart)
+    stop
+    start
+    ;;
+  *)
+    echo "Usage: $0 {start|stop|restart|uninstall}"
+esac    
+----
+  * Add executable flag on the file:
+
+  chmod +x /etc/init.d/rcdcap
+ 
+  * Enable rcdcap service
+
+  sudo update-rc.d rcdcap defaults 18 02
+
+  * Start service
+
+  sudo /etc/init.d/rcdcap start
+
+  * Rerun Security Onion wizard to configure tweak mon1 interface and include it to be monitored. Attention: network configuration file will be overwritten. Backup if needed, or skip that part in the wizard and adapt manually.
```

Comment:
Not sure we can say `Attention: blablabla` in english. Most likely should be `Warning: Zammits cousin laundry service uses no name javel on your clothes`

And if you put `WARNING: Do not consume Zammits sandwiches if pregnant` on its on line its going to format it nicely when generating our doc

Just like magic but less spectacular.